cypress/support/commands.ts

```diff
@@ -886,7 +904,7 @@ export const selectScriptFieldOrTag = (name: string, beActive: boolean) => {
 }
 
 export const switchToDataExplorer = (typ: 'new' | 'old') => {
-  cy.getByTestID('page-title').contains('Data Explorer')
+  cy.getByTestID('data-explorer-page').should('exist')
```

Comment:
Why do you use `exist` instead of `be.visible`?

Reply:
Good question. We could done either way. It's a bigger deal to use the `is.visible` for clickable elements like buttons etc --> to make sure the user can interact with the element.

Here, we are just looking for the page loading having finished (future proofing, for whatever cypress test calls this fixture). So we could use either visible, or exist.